packages/graphrag-storage/graphrag_storage/storage_type.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2024 Microsoft Corporation.
+# Licensed under the MIT License
+
+
+"""Builtin storage implementation types."""
+
+from enum import StrEnum
+
+
+class StorageType(StrEnum):
+    """Enum for storage types."""
+
+    FILE = "file"
```

Comment:
I think we've been generally moving to TitleCase, which would match cross-platform stuff a little more too.

File
Memory
AzureBlob
AzureCosmos